src/app/shared/entry.ts

```diff
@@ -412,11 +412,14 @@ export abstract class Entry<V extends WorkflowVersion | Tag> implements OnDestro
    * Will decode the URL
    * @return {void}
    */
-  decodeURL(type: string): void {
-    const url = decodeURIComponent(window.location.href);
-    const containersIndex = this.getIndexInURL('/' + type);
-    const newPath = url.substring(containersIndex);
-    this.location.replaceState(newPath);
+  decodeURL(): void {
+    let url = decodeURIComponent(window.location.href);
```

Comment:
Can this be `const url = decodeURIComponent(window.location.pathname);`, then you don't need the lines 417-421?